extensions/who-is-visiting-this-content/app.R

```diff
@@ -125,24 +246,31 @@ server <- function(input, output, session) {
     get_content(client)
   }) |> bindCache("static_key")
 
-  user_names <- reactive({
+  users <- reactive({
     get_users(client) |>
-      mutate(full_name = paste(first_name, last_name)) |>
-      select(user_guid = guid, full_name, username)
+      mutate(
+        full_name = paste(first_name, last_name),
+        display_name = paste0(full_name, " (", username, ")")
+      ) |>
+      select(user_guid = guid, full_name, username, display_name, email)
   }) |> bindCache("static_key")
 
-  usage_data <- reactive({
+  firehose_usage_data <- reactive({
     get_usage(
       client,
-      from = date_range()$from_date,
-      to = date_range()$to_date + hours(23) + minutes(59) + seconds(59)
+      from = as.POSIXct(date_range()[1]),
+      to = as.POSIXct(date_range()[2]) + hours(23) + minutes(59) + seconds(59)
```

Comment:
Any reason to _not_ use the names here? It's mostly obvious, but `from_date` and `to_date` make it super clear as opposed to `1` and `2`

Reply:
The switch from names to indices happened because I switched from my toy fixed date range to wrapping the `dateRangeInput()`, which just provides indices.

But I think that since we're wrapping it anyway, we could make the values named to make it clearer elsewhere in the code. :)